packages/cli/src/commands/console.js

```diff
@@ -0,0 +1,34 @@
+import path from 'path'
+import repl from 'repl'
+
+import { getPaths } from 'src/lib'
+
+/**
+ * Console is less technical and more Railsy
+ */
+export const command = 'console'
+/**
+ * @todo
+ * Appropriate alias?
+ */
+export const aliases = ['c']
+
+/**
+ * @todo
+ * Should we make this more console related?
+ * I.e. does using the word REPL here contradict our efforts to be less technical?
+ */
+export const description = 'Start the Redwood REPL'
+
+const mapDBToContext = (ctx) => {
+  const { db } = require(path.join(getPaths().api.lib, 'db'))
+  ctx.db = db
+}
+
+export const handler = () => {
+  // Transpile on the fly
+  require('@babel/register')
```

Comment:
This should probably import the user's babel configuration. We're doing the same thing in the dev-server:

https://github.com/redwoodjs/redwood/blob/558ecee62a70cf7169eae3bc076cd20b6f5fd044/packages%2Fdev-server%2Fsrc%2FwatchApiSide.ts#L52-L58

Reply:
Done!